woocommerce/woocommerce-ios
# Shipping Labels: address formatting in the Shipping Label Form and loader indicator in the Address Validation Form

Part of #2973 

## Description
In this PR I applied some minor fixes, plus some improvements to the shipping label form. In particular, in this PR now the formatted address will be shown in the Shipping Label Form, for the origin and destination address. Also, the done button for validating the address now the done button will be transformed in a loader indicator during the address validation request.

## Changes
- Added the new property `isTrivialNormalization` in `ShippingLabelAddressValidationResponse` entity because it will be useful in the near future for implementing the possibility for the user to choose the suggested address.
- Implemented an extension of the `ShippingLabelAddress` entity, where a method returns the `name`, `company`, and `address`. Basically combines the various components of a ShippingLabelAddress.
- Added a completion block to `ShippingLabelAddressFormViewController` that will return the latest shipping label address updated to the previous view controller.
- Now in the `ShippingLabelAddressFormViewController` the done button will be transformed in a loader indicator during the address validation request.
- Fixed the logic of the top banner view that was wrong.
- Fixed the keyboard type for some fields in the shipping label address form.
- Fixed the state for the `address` field, which now can be in the error state.
- Showed the formatted address (origin and destination) in the Shipping Label Form.
- Handle origin and destination address value changes in `ShippingLabelFormViewModel` + tests.

## Testing
1. Launch the app in debug mode (because of the feature flag).
2. Open an order detail.
3. Press the button "Create Shipping Label".
4. In the Shipping Label Form -> you should see the formatted address for the origin and the destination address.
5. Press the button "Continue" in the Shipping Label form (under the origin address).
6. Try to validate an address pressing the Done button on top. -> The done button should be transformed in a loader indicator during the address validation request.
7. Try to edit the various field of the address. -> Make sure that every field has the right keyboard type (standard or numeric pad).


Update release notes:
